docs/integrations/gitops/github.mdx

```diff
@@ -83,10 +83,13 @@ The private key is at the location you chose and the public key is at the same l
     ```shell
     curl -X POST "https://api.permit.io/v2/projects/<your-project-id-or-key>/repos" -H "Content-Type: application/json" -H "Authorization: Bearer {API_KEY from step 4}" --data-binary @"path/to/your/json"
     ```
-10. Go back to your GitHub repository, you should see new branches, one for each environment in your project.
-11. After validating your new branches policy, you need to activate the new repository in permit,
-to do this, replace the placeholder with your project id or key and execute a PUT request to Permit:
+10. Go back to your GitHub repository, you should now see new branches, one for each environment in your project. the branches are named after the environment id in the following format: `permit/generated/<env_id>`.
+11. After validating your new policy branches, you need to activate the new repository in Permit,
+to do this, replace the below placeholders with your project id or key and execute a PUT request to Permit:
     ```shell
     curl -X PUT "https://api.permit.io/v2/projects/<your-project-id-or-key>/repos/<your-repo-id-or-key>/activate" -H "Authorization: <your-permit-api-key>"
     ```
-12. Your git repository is now used for your PDPs policy 🚀
\ No newline at end of file
+12. To add custom policy code for each env, you first need to checkout the right branch `permit/generated/<env_id>`. Within the branch  be sure only add your policy code inside the `custom` folder. You can either add your custom Rego code to the `custom/root.rego` file or add another file and import it in the `custom/root.rego` file (if you edit any policy code outside this folder it can be override by auto generared code / break the autogenerated code)
```

Comment:
I think that it should be "it can be overwritten" 
I'm not sure so check me

Reply:
Or, reviewed the English :) 
I wanted from you the tech side 👍

Reply:
> I think that it should be "it can be overwritten" I'm not sure so check me

It can be both. It overwrites the content of the files, and as a result it overrides their intended changes.